release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -1,20 +1,16 @@
 ## 2022-07-07 Version 2.1.0 Release Notes
 
-
-### Breaking Changes in 2.1.0
-
-#### Remove Mapping types
+### Remove Mapping types
```

Comment:
This should not be a category but rather under some category. More details: https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md
same  applied to all below major headings (upgrades, deprecations)

Reply:
Got it. Let me change their categories

Reply:
If this is not a breaking change can we please put it to either bug fixes or enhancements?

Reply:
Yes that makes sense